packages/dds/merge-tree/api-report/merge-tree.legacy.alpha.api.md

```diff
@@ -641,6 +641,24 @@ export interface MergeTreeRevertibleDriver {
     removeRange(start: number, end: number): void;
 }
 
+// @alpha (undocumented)
+export interface ObliterateInfo {
```

Comment:
SegmentGroup is deprecated, so can we deprecate this as well to discourage new usage?

Reply:
i had the same thought, but decided against it as the suppressions are a bit annoying, but will make the change, as it feels a bit safer even if it is odd to add a type that is directly deprecated :)